### DISCUSSION TITLE
Alanwar TV

### DISCUSSION THREAD

**Discussion by Mennaruuk** [iptv-org/community#687] [Q&A]
**Channel Name:** Al-Anwar TV
**Channel ID:** AlanwarTV.kw
**Country:** UK
**Language:** Arabic
**Website:** https://al-anwar.tv
**Stream URL (optional):** xxx
**Notes (optional):** Notoriously difficult throughout the years to find a stream link for this channel. Also, [site was likely seized in error last month](https://theintercept.com/2021/06/26/us-iran-censor-websites-evidence) by the U.S. government, which probably mistaken it for another channel, [Al-Anwar 2](https://anwartv2.com).

**Comment by BellezaEmporium** [comment]
Site "https://al-anwar.tv" was seized by the US gov, indeed. It's not going to be easy to get something from that channel.

**Comment by MapGuy11** [comment]
Here is there old stream url: 
http://199.167.148.85:1935/alanwar/alanwar/playlist.m3u8
http://iptvsng.dyndns.org:8080/live/Alanwartv/LIVEANWAR/3.m3u8

**Comment by Alstruit** [comment]
https://streamboxtv.net:25461/live/ALANWAR/TV/231.m3u8
Can't tell if it's online or not.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19726
